README.md

```diff
@@ -18,6 +18,14 @@ writing tests, checking results and automated testing in Ruby.
 test-unit ships as part of Ruby as a bundled gem. To check which version is included, see https://stdgems.org/test-unit/.
 It's only necessary to install the gem if you need a newer version.
 
+test-unit is one of two testing unit libraries bundled with Ruby.
+The other is [minitest](https://github.com/seattlerb/minitest), which is newer, but test-unit is still active.
```

Comment:
Should we mention that minitest was started after test-unit explicitly? Is it important for users?

I feel that this wording is negative for test-unit.,,

Reply:
Yes, I see, how about:

```suggestion
test-unit is the original Ruby unit testing library, and is still active.
It is one of two testing unit libraries bundled with Ruby, the other being [minitest] (https://github.com/seattlerb/minitest),
```

Reply:
It's better than before. Thanks.

"testing unit" -> "unit testing"

Reply:
@kou thanks, updated.